### PR TITLE
[GR-1217] Replace BamQC's TotalReads with FastQC's TotalSequences in Single-Lane Reports

### PR DESCRIPTION
**Pull request by avarsava**
Holding off on Call-Ready reports until merging the data is figured out.

Savo: please try different merge types and let me know how you feel about displaying FastQC data before it can be merged with BamQC, as discussed in our meeting.

### CODE REVIEW

File: application/dash_application/views/single_lane_ts.py [hunk 1]
```diff
@@ -88,16 +90,18 @@
 
 def get_bamqc_data():
     bamqc_df = util.get_bamqc3_and_4()
+    bamqc_df = util.df_with_fastqc_data(bamqc_df, [BAMQC_COL.Run, BAMQC_COL.Lane, BAMQC_COL.Barcodes])
+
     bamqc_df[special_cols["Total Reads (Passed Filter)"]] = round(
         bamqc_df[BAMQC_COL.TotalReads] / 1e6, 3)
```
Comment (slazicoicr) [inline application/dash_application/views/single_lane_ts.py]: Stray `BAMQC_COL`

**Review by slazicoicr — COMMENTED** [review]
Still several uses of `BAMQC_COL.TotalReads` in single lane TS.

**Comment by avarsava** [comment]
> Still several uses of `BAMQC_COL.TotalReads` in single lane TS.

Rolled back one commit too far when undoing work on Call-Ready pages. Fixed.

File: application/dash_application/views/single_lane_ts.py (hunk 1 above)
Comment (avarsava) [reply application/dash_application/views/single_lane_ts.py]: Unsure what you mean by this. Those are the columns to merge on.

Comment (avarsava) [reply application/dash_application/views/single_lane_ts.py]: Fixed

**Review by slazicoicr — APPROVED** [review]

**Review by apmasell — APPROVED** [review]

**Pull request merged**